pagopa/io-app
# chore(IT Wallet): [SIW-1447] Allow user to cancel CieID identification

> [!WARNING]  
> This PR depends on #6128

## Short description
This PR allows a user to cancel the CieID identification after the external browser was opened, thus avoiding the loading screen that can't be dismissed before 120 seconds.

## List of changes proposed in this pull request
- Every identification mode now accepts an optional [AbortController](https://developer.mozilla.org/en-US/docs/Web/API/AbortController)
- Added `abort` event and `isOperationAborted` guard
- Added `ItwIssuanceLoadingScreen` component to handle the cancel button logic during loading
- Added new copy when waiting for CieID identification

## How to test
> [!NOTE]  
> Needs testing on iOS.

1. Start the eID issuance flow with CieID
2. The cancel button should not be visible before the browser app is opened
3. Go back to the app after the browser was opened, you should see a cancel button and should be able to abort the operation
4. Try the other identification modes as well for regressions

https://github.com/user-attachments/assets/0e18c838-f371-4023-84a2-9d22b60f3493
